### PR TITLE
Add redbox test for 2048

### PR DESCRIPTION
**Pull request by vjeux**
This is just a dummy test to make sure that the app renders something correctly. This will help us catch a lot of integration errors that we had. This app was broken for more than a month without anyone noticing on the team :(

### CODE REVIEW

File: Examples/2048/2048 Tests/_048_Tests.m [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ */
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+#import "RCTLog.h"
+#import "RCTRootView.h"
+
+#define TIMEOUT_SECONDS 240
+#define TEST_ID_TO_LOOK_FOR @"main"
```
Comment (vjeux) [inline Examples/2048/2048 Tests/_048_Tests.m]: I copied this entire file from SampleAppTest. cc @sahrens @frantic

File: Examples/2048/Game2048.js [hunk 2]
```diff
@@ -197,6 +197,7 @@ class Game2048 extends React.Component {
 
     return (
       <View
+        testID="main"
```
Comment (vjeux) [inline Examples/2048/Game2048.js]: making great use of testID :)

File: scripts/objc-test.sh [hunk 3]
```diff
@@ -26,7 +26,16 @@ trap cleanup EXIT
...
+  -scheme 2048 \
+  -sdk iphonesimulator \
+  test
+
 xctool \
   -project Examples/UIExplorer/UIExplorer.xcodeproj \
-  -scheme UIExplorer -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 5,OS=8.3' \
+  -scheme UIExplorer \
+  -sdk iphonesimulator \
+  -destination 'platform=iOS Simulator,name=iPhone 5' \
```
Comment (vjeux) [inline scripts/objc-test.sh]: OS=8.3 didn't work on my machine. Let's try to remove it and see what travis says ...

**Comment by sahrens** [comment]
Need to add a scheme I guess?

> ERROR: Cannot find schemes. Please consider creating shared schemes in Xcode.
TIP: This might happen if you're relying on Xcode to autocreate your schemes
and your scheme files don't yet exist.  xctool, like xcodebuild, isn't able to
automatically create schemes.  We recommend disabling "Autocreate schemes"
in your workspace/project, making sure your existing schemes are marked as
"Shared", and making sure they're checked into source control.

**Comment by sahrens** [comment]
Otherwise looks fine - go ahead and import the diff internally.

**Comment by vjeux** [comment]
Yeah I need to figure out what incantation i need to do to get a scheme up and running. I tried locally and it worked fine

**Pull request closed without merging**